app/api/db/models/__init__.py

```diff
@@ -1 +1 @@
-from . import eligibility_models  # noqa: F401
+from . import example_models  # noqa: F401
```

Comment:
nit: don't need to ignore F401, the correct way to handle this is to define `__all__`
```suggestion
from . import example_models

__all__ = "example_models"
```